src/server/api/routes/api-router.js

```diff
@@ -7,6 +7,7 @@ const modulesRouter = require('./modules.router');
 
 // messages router imports
 const messagesRouter = require('./messages-router');
+const userRouter = require('./userDeleteById-router');
```

Comment:
The user router has already been added, merge develop into your branch. The delete user should lay under the user router.